SOURCES/0002-use_new_api.patch

```diff
@@ -0,0 +1,14 @@
+--- a/src/r8125.h	2022-03-31 07:27:49.000000000 +0000
++++ b/src/r8125.h	2022-05-06 02:36:55.000000000 +0000
+@@ -51,11 +51,9 @@
+ typedef int netdev_tx_t;
+ #endif
+ 
+-/*
+ #if LINUX_VERSION_CODE >= KERNEL_VERSION(4,12,0)&& !defined(ENABLE_LIB_SUPPORT)
+ #define RTL_USE_NEW_INTR_API
+ #endif
+-*/
```

Comment:
What does this change, and why is it needed?

Reply:
How does commenting out the whole 3 lines enable `RTL_USE_NEW_INTR_API`? As I read it, the condition is already true: kernel version is >= 4.12 and ENABLE_LIB_SUPPORT is not defined.

Reply:
Because it's uncommenting the lines and enabling them.